### PR TITLE
RA-01 User model

### PR DESCRIPTION
**Pull request by BigiDigi89**
Added user model with devise

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```
Comment (deHelden) [inline app/models/user.rb]: Shouldn't we define :omniauthable for future pupuses, should we?

File: app/views/devise/confirmations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (deHelden) [inline app/views/devise/confirmations/new.html.erb]: Please convert devise views to slim 
https://github.com/slim-template/html2slim

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.string "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string "unconfirmed_email"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "role", default: "student"
```
Comment (deHelden) [inline db/schema.rb]: Role should be a reference(at least integer) to table user_roles

Comment (lincAlinc) [reply db/schema.rb]: @deHelden. we agreed not to make a separate table for roles

File: app/models/user.rb (hunk 1 above)
Comment (BigiDigi89) [reply app/models/user.rb]: I think too early for that. We can easily do this in the future. While in the requirements it was not said

File: config/initializers/devise.rb [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '76119bd93113fd14c188a1ee276057b21fe8d71d57a11d6f3fe3fba6ad45b690b1db91a88766803b5d97ef04a6206704d44ca2f8404c7f0d4d6e9cc3b0a8a200'
+
```
Comment (krlklim) [inline config/initializers/devise.rb]: Remove trailing whitespace.

**Pull request merged**